src/components/ChangePassword.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+    <section class="vh-100 bg-light">
+        <div class="container py-5 h-100">
+            <div class="row d-flex justify-content-center align-items-center h-100">
+                <div class="col-12 col-md-8 col-lg-6 col-xl-5">
+                    <div class="card shadow-2-strong" style="border-radius: 1rem;">
+                        <div class="card-body p-5 text-center">
+                            <form @submit.prevent="changePassword()">
+                                <h3 class="mb-5">Change Password</h3>
+                                <div class="form-outline mb-4">
+                                    <input type="password" class="form-control form-control-lg" v-model="oldPassword" />
+                                    <label class="form-label">Old Password</label>
+                                </div>
+                                <div class="form-outline mb-4">
```

Comment:
Make another field called confirm password which must have same value as new password in order for FE validation to succeed and to send request on BE.